docs/SHOPIFY-SYNC.md

```
@@ -355,3 +355,138 @@ theme's existing hero buttons:
+Live rules today carry `".indexOn": ["state"]`. **Add `"updatedAt"`:**
+
+```json
+"shopify_publish": {
+  ".indexOn": ["state", "updatedAt"]
+}
+```
+
+**RTDB does not sort an unindexed query — it refuses it.** Verified against the
+live database on 3 Sep 2026:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)